RecoLocalCalo/HcalRecAlgos/interface/PedestalSub.h

```diff
@@ -0,0 +1,31 @@
+#ifndef PedestalSub_h
```

Comment:
The file include guard should be specific to the file and package
e.g.,  RecoLocalCalo_HcalRecAlgos_PedestalSub_h